src/mixins/payment-mixin.ts

```diff
@@ -129,11 +132,11 @@ export class PaymentMixin extends Mixins(ActionMixin) {
     // Any additional data supplied here, eg. supplying
     // a businessInfo object, will override values found
     // in the corresponding Name Request
-    const data = {
-      // Comment this out to use direct pay
-      /* paymentInfo: {
-        methodOfPayment: methodOfPayment
-      }, */
+    const data: any = {
+      // Comment this out to use direct pay:
+      // paymentInfo: {
+      //   methodOfPayment: methodOfPayment
+      // },
```

Comment:
Is this for local testing or meant to be left in?

Reply:
This was there before. I only changed the comment style.
I'm not afraid to delete obsolete code, but this code looks like we might use it some day.

Reply:
Sounds good